modules/solid_mechanics/src/materials/RadialReturnStressUpdate.C

```diff
@@ -250,21 +261,32 @@ RadialReturnStressUpdateTempl<is_ad>::updateState(
   // configuration
   GenericRankTwoTensor<is_ad> deviatoric_trial_stress = stress_new.deviatoric();
 
-  // compute the effective trial stress
+  // compute the effective trial stress.  Ternary because the sqrt is not differentiable at 0
   GenericReal<is_ad> dev_trial_stress_squared =
       deviatoric_trial_stress.doubleContraction(deviatoric_trial_stress);
   GenericReal<is_ad> effective_trial_stress = MetaPhysicL::raw_value(dev_trial_stress_squared)
                                                   ? std::sqrt(3.0 / 2.0 * dev_trial_stress_squared)
                                                   : 0.0;
+  // Initialize models around a stress that uses the previous inelastic strain increment to
+  // predict the final stress state for this step. (Dunne & Petrinic eq 5.16 )
+  // fixme three shear mod is set in comptueStressInitialize
```

Comment:
_three_shear_modulus is set in RadialReturnStressUpdate::computeStressInitialize.  We then call RadialReturnStressUpdate::computeStressInitialize from all of the derived computeStressInitialize and we check that we called it in the Assert on ln 284. 
 Why don't we just set it in this function before we call computeStressInitialize?  Then make RadialReturnStressUpdate::computeStressInitialize a no-op function and not call it from any derived classes.